prometheus.php

```diff
@@ -10,6 +10,10 @@
 use Automattic\VIP\Prometheus\Potential_Multi_Dataset_Queries_Collector;
 use Automattic\VIP\Prometheus\Multisite_Stats_Collector;
 // @codeCoverageIgnoreStart -- this file is loaded before tests start
+if ( extension_loaded( 'newrelic' ) ) { // Ensure PHP agent is available
+	newrelic_ignore_transaction();
+}
+
```

Comment:
Same as the above, but the endpoint is here: 

https://github.com/Automattic/vip-go-mu-plugins/blob/develop/prometheus/endpoint.php